compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -194,21 +195,21 @@ pub fn unsize_ptr<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
     src_ty: Ty<'tcx>,
     dst_ty: Ty<'tcx>,
     old_info: Option<Bx::Value>,
-) -> (Bx::Value, Bx::Value) {
+) -> OperandValue<Bx::Value> {
```

Comment:
This will need to be implemented in cg_clif too. I can do this myself once this PR is merged.